pkgs/development/python-modules/vllm/default.nix

```diff
@@ -123,29 +131,38 @@ buildPythonPackage rec {
 
   propagatedBuildInputs =
     [
-      psutil
-      ray
+      aioprometheus
+      fastapi
+      lm-format-enforcer
+      numpy
+      openai
+      outlines
       pandas
+      prometheus-fastapi-instrumentator
+      psutil
+      py-cpuinfo
       pyarrow
+      pydantic
+      pyzmq
+      ray
       sentencepiece
-      numpy
+      tiktoken
       torch
+      torchvision
       transformers
-      outlines
-      xformers
-      fastapi
       uvicorn
-      pydantic
-      aioprometheus
+      xformers
     ]
     ++ uvicorn.optional-dependencies.standard
     ++ aioprometheus.optional-dependencies.starlette
     ++ lib.optionals cudaSupport [
-      pynvml
       cupy
+      pynvml
     ];
 
-  stdenv = if cudaSupport then cudaPackages.backendStdenv else stdenv_pkg;
+  dontUseCmakeConfigure = true;
```

Comment:
Nit: this should go before `configurePhase` and respective hooks; side note: this `default.nix` also puts `*Inputs` after configure/build phases which is unconventional for nixpkgs and so less convenient to read

Reply:
this default.nix doesn't have a configurePhase, should it go above the Inputs?

Reply:
I think we oscillate somewhere around

```
pname
version
pyproject
disabled
src
patches
postPatch
[native]BuildInputs
dontUseCmakeConfigure
configurePhase
buildPhase
checkPhase
installPhase
postInstallPhase # the real install phase in case of python
```

Reply:
looking at this list I still don't really see what's out of place, we have
```
pname
version
src
stdenv
<env var>
postPatch
preBuild
*buildInputs
dontUseCmakeConfigure
relaxDeps
importsCheck
meta
```

is it that preBuild belongs before the (missing) buildPhase? So between dontUseCmakeConfigure and relaxDeps?